tests/test.c

```diff
@@ -173,7 +173,7 @@ struct Test tests[] = {
     TEST_CASE_EXPECTED(long_atoms, 4),
     TEST_CASE_EXPECTED(test_concat_badarg, 4),
     TEST_CASE_EXPECTED(register_and_whereis_badarg, 333),
-    TEST_CASE_EXPECTED(test_send, -3),
+    TEST_CASE_EXPECTED(test_send, -5),
```

Comment:
Could we please have modified tests that return 0? Then we can use the simplified macro TEST_CASE.

With stacktraces, tests that return integer values are less useful.

Reply:
Certainly! I overlooked the TEST_CASE macro, and was just making the minimal changes necessary to test the added functionality.